clients/cobol-lsp-vscode-extension/src/test/suite/lsp.spec.test.ts

```diff
@@ -529,11 +529,11 @@ suite("Integration Test Suite", function () {
 
   test("Show errors only for opened files", async () => {
     // Open program with error inside a copybook
-    await helper.showDocument("TESTCPY1.cbl");
-    const progUri = await helper.getUri("TESTCPY1.cbl");
+    let editor = await helper.showDocument("TESTCPY1.cbl");
+    const progUri = editor.document.uri;
```

Comment:
VsCode API has a different behavior for below 2 cases
`vscode.languages.getDiagnostics(vscode.Uri.file("c:/Users/TESTCPY1.cbl"))`   <-- SHOWS DIAGNOSTICS
`vscode.languages.getDiagnostics(vscode.Uri.file("//./c:/Users/TESTCPY1.cbl")) `<-- NO DIAGNOSTICS

Reply:
I think the test is not opening the UNC path.
```
path.join(__dirname, "../../../../tests/test_files/project")
```
would need to be prefixed in the `launchArgs` with `\\.\`

Reply:
I tried something in the same line [here ](https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/daadceac5f2c350b9f5cd13485c01256640978fb/clients/cobol-lsp-vscode-extension/src/test/suite/testHelper.ts#L73)

I see paths being passed as `file:///./c%3A/Users/che-che4z-lsp-for-cobol/tests/test_files/project/TESTCPY1.cbl`, when opening the document